### DISCUSSION TITLE
Build Issue

### DISCUSSION THREAD

**Discussion by MOAMaster** [audacity/audacity#7543] [Development]
### Description

I opened the audacity folder in visual studio 2019. I asked to build. A lot built correctly. But, in the end, the build failed with the following error: Severity	Code	Description	Project	File	Line	Suppression State
Error		cmake version 3.20.21032501-MSVC_2	E:\Audacity\audacity-master\out\build\x64-Debug\audacity-master	E:\Audacity\audacity-master\out\build\x64-Debug\EXEC	1	
What do I do?
Thanks.
Bruce Krell

**Comment by vsverchinsky** [comment]
Hello Bruce,
Could you please try updating your CMake to more recent version?

**Comment by MOAMaster** [comment]
I installed the latest version 3.30, then edited the cmakeSettings.json to point to this.

I still get an error message, 
Severity	Code	Description	Project	File	Line	Suppression State
Error		cmake version 3.30.0	E:\Audacity\audacity-master\out\build\x64-Debug\audacity-master	E:\Audacity\audacity-master\out\build\x64-Debug\EXEC	1	

An exe is created. I tried to run this. But get a bunch of missing DLLs.

Truthfully, all I want is to create a DLL with access to some of the effects, specifically your EQ.
I use this a lot and want to access this from a client program that applies the same set of EQ settings
to a list of files. That would make my workload a lot easier to handle.

Anyhow, can't seem to get Audacity itself to build, even with the latest version of cmake.

Any help to get Audacity to build will be appreciated.

Bruce
